### PR TITLE
Resolves 25 Rubocop Style/SafeNavigation conflicts

### PR DESCRIPTION
**Pull request by wburns84**
Rubocop: 433 files inspected, 471 offenses detected
These are fixed by https://github.com/rubyforgood/openstreetmap-website/pull/8

Finished in 211.496735s, 5.1821 runs/s, 1618.2945 assertions/s.
1096 runs, 342264 assertions, 0 failures, 0 errors, 0 skips

### CODE REVIEW

File: app/controllers/amf_controller.rb [hunk 1]
```diff
@@ -883,12 +883,10 @@ def tags_ok(tags)
   # in the +tags+ hash.
   def strip_non_xml_chars(tags)
     new_tags = {}
-    unless tags.nil?
-      tags.each do |k, v|
-        new_k = k.delete "\000-\037\ufffe\uffff", "^\011\012\015"
-        new_v = v.delete "\000-\037\ufffe\uffff", "^\011\012\015"
-        new_tags[new_k] = new_v
-      end
+    tags&.each do |k, v|
```
Comment (benreyn) [inline app/controllers/amf_controller.rb:886]: This is the only one of these that I feel itchy scratchy about... Do we know what this code will do if `tags` is nil? Perhaps this one is worth leaving alone.

Comment (wburns84) [reply app/controllers/amf_controller.rb:886]: It behaves the same:

```
irb(main):002:0> nil&.each
=> nil
```

Comment (wburns84) [reply app/controllers/amf_controller.rb:886]: I agree it's odd looking.  We can use a guard clause instead.

Comment (benreyn) [reply app/controllers/amf_controller.rb:886]: Hmm, lets leave it as is for now and see how this ends up looking when it makes its way upstream. Thanks for checking!

**Pull request merged**